src/searchbar.vue

```diff
@@ -0,0 +1,52 @@
+<script>
+    import Typeahead from './Typeahead.vue'
+    export default {
+      extends: Typeahead,
+      computed: {
+        primitiveData() {
+          if (this.data) {
+            var matches = []
+            var words
```

Comment:
Remove this declaration, declare this below when it is initialized, so it is something like:

```js
const words = keywords ? keywords + title : title
```